cypress/tests/bucket-policy-spec.ts

```diff
@@ -0,0 +1,92 @@
+import { DATA_FEDERATION_NAMESPACE, SECOND, MINUTE } from '../constants/common';
+import { dataSourceNSFS } from '../mocks/data-source';
+import { MCGMSCommon, projectNameSpace } from '../views/common';
+import { pvc } from '../views/pvc';
+
+describe('Bucket policy page', () => {
+  const singleBucketPolicy = 'e2e-bucket-single';
```

Comment:
these are all constant names, should we write them in caps? what is your opinion?

Reply:
or we can have a test value file form where all values will be picked up

Reply:
done...